.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+test_task:
+  alias: "build_and_test"
+  name: "Build and Test"
+  container:
+    image: docker.io/library/rust
```

Comment:
This will break if/when the pull rate limit is exceeded.  A new quay.io namespace should probably be created for netavark, and this image copied there manually.  This will also prevent sudden surprises by unexpected changes to the image (this happened before with the registry image).